.github/workflows/reviewdog.yml

```diff
@@ -69,3 +69,5 @@ jobs:
           reporter: github-check
           level: warning
           locale: "US"
+
+# TODO_IMPROVE: Enforce using k.Logger() when logging in the `x/` directory code.
```

Comment:
IMHO the cosmos-sdk logger is not ideal because its debug level is not really useful; the default level is `info` and the `debug` log contains a bunch of cosmos-sdk specific output which obscures our application's debug logs. I think we may be better served by using polylog on-chain, for example.

```suggestion
# TODO_IMPROVE: Consider whether to enforce using a particular logger when logging in the `x/` directory code. 
```

Reply:
This change and discussion should probably be moved to the `migration/base` review.